openapi-specs/schema.json

```diff
@@ -4801,6 +4829,24 @@
             "schema": {
               "$ref": "#/definitions/BackupCreateRequest"
             }
+          },
+          {
+            "name": "X-Aws-Access-Key",
+            "in": "header",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "X-Aws-Secret-Key",
+            "in": "header",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "X-Aws-Session-Token",
+            "in": "header",
+            "required": false,
+            "type": "string"
```

Comment:
I think that we shouldn't add header definitions here in schema, there's lots of other headers that we use beside the AWS ones, so I would not include this information here